integrity-backend/input-archiveweb-examples/3e11cc57daf3bad8375935cad4878123acc8d769551ff90f1b1bb0dc597-meta-content.json

```
@@ -0,0 +1,35 @@
+{
+  "contentMetadata":
+  {
+    "name": "My Archive",
+    "mime": "application/wacz",
+    "description": "Web archive uploaded to Dropbox",
+    "author":
+    {
+      "@type": "Organization",
+      "identifier": "https://starlinglab.org",
+      "name": "Starling Lab"
+    },
+    "dateCreated": "2022-04-14T01:27:46Z",
+    "extras":
+    {
+      "localsignSoftware": "Webrecorder ArchiveWeb.page 0.7.8, using warcio.js 1.5.0",
+      "localsignPublicKey": "MHYwEAYHKoZIzj0CAQYFK4EEACIDYgAEoCCciTZZ2Km8TGGsl2s4c91CyIPEVFcxyU6LEbRgnQtiuRtugKGmyxLKpK41kPsiDAOvmxA74VMuFAhiaqE3FfdSAPdnM8+V2ugYY+uNHlQlVIakJnuNZx/AYj5AoknV",
+      "localsignSignature": "9AHneB8LTEP80DNXI997A5f4j2//rRW911t1scyjnxBoYJut3afw6quK1Tww+DdwcgVqlNjfVMtH4rUN1v2VuOgq5bidXpoulInYQR7oQAJe4Tlq8H9gt84pcb8oRP3I",
+      "waczVersion": "1.1.1",
```

Going to extract `title` from the `wacz` file so that it can be used in the name. Its easier if its passed in as `extras` so i dont have to hold it as a different variable some where else.

```suggestion
      "waczVersion": "1.1.1",
      "waczTitle": "Title of file",
```

---

Used for `name`